src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -623,6 +642,56 @@ def __print_restore_preview(kvs_to_restore, kvs_to_modify, kvs_to_delete):
     return True
 
 
+def __is_json_content_type(content_type):
+    # pylint: disable=too-many-return-statements
+    if not content_type:
+        return False
+
+    content_type = content_type.strip().lower()
+    mime_type = content_type.split(';')[0].strip() if ';' in content_type else content_type
+    type_parts = mime_type.split('/') if '/' in mime_type else None
+
+    if not type_parts or len(type_parts) != 2:
+        return False
+
+    (main_type, sub_type) = type_parts
+    if main_type != "application":
+        return False
+    if "json" not in sub_type:
+        return False
+
+    sub_types = sub_type.split('+') if '+' in sub_type else None
+
```

Comment:
What should be the valid MIME json type? application/xxxx+json is valid then how about application/json+xxxx or application/xxx+json+xxxx

Reply:
Yes, both "application/json+xxxx" and "application/xxx+json+xxxx " are also valid JSON.

---
In reply to: [448782461](https://github.com/Azure/azure-cli/pull/14170#discussion_r448782461) [](ancestors = 448782461)